src/bun.js/webcore/body.zig

```diff
@@ -370,11 +371,20 @@ pub const Body = struct {
             }
         }
 
+        fn _lockedSize(this: *const Value) Blob.SizeType {
+            if (this.Locked.readable) |readable| {
+                return readable.ptr.Bytes.size_hint;
+            }
+
+            return this.Locked.size_hint;
+        }
+
         pub fn size(this: *const Value) Blob.SizeType {
             return switch (this.*) {
                 .Blob => this.Blob.size,
                 .InternalBlob => @as(Blob.SizeType, @truncate(this.InternalBlob.sliceConst().len)),
                 .WTFStringImpl => @as(Blob.SizeType, @truncate(this.WTFStringImpl.utf8ByteLength())),
+                .Locked => this._lockedSize(),
```

Comment:
Can you add a comment explaining why this is safe? It's not clear from reading this if `_lockedSize` returns the currently buffered size (which may not be the entire size in the case of chunked transfer encoding) or if it returns the whole size. It's also unclear what happens when the size is not known.

This method has to be accurate or else the HTTP response will potentially hang/timeout. So if we only return the estimated size, it's incorrect.